rest-api/test/unit_test/offline_test/genomic_set_file_handler_test.py

```diff
@@ -5,9 +5,9 @@
 import pytz
```

Comment:
If we're moving this from offline to rest-api does it need to be an offline_test ?

Reply:
Done